gp/node.py

```diff
@@ -22,14 +28,17 @@ class Node:
     """Base class for primitive functions used in Cartesian computational graphs.
     """
 
-    _arity = None
-    _active = False
-    _inputs = None
-    _output = None
-    _idx = None
-    _is_parameter = False
-
-    def __init__(self, idx, inputs):
+    __name__ = "Node"
```

Comment:
why do we need this?

Reply:
Don't remember now, but I added it because of some mypy complaint. Will look into it.

Reply:
removed.